src/main/java/com/sap/sgs/phosphor/fosstars/data/interactive/AskAboutSecurityTeam.java

```diff
@@ -0,0 +1,37 @@
+package com.sap.sgs.phosphor.fosstars.data.interactive;
+
+import static com.sap.sgs.phosphor.fosstars.model.feature.oss.OssFeatures.HAS_SECURITY_TEAM;
+
+import com.sap.sgs.phosphor.fosstars.model.ValueSet;
+import com.sap.sgs.phosphor.fosstars.model.value.BooleanValue;
+import com.sap.sgs.phosphor.fosstars.tool.YesNoSkipQuestion;
+import com.sap.sgs.phosphor.fosstars.tool.YesNoSkipQuestion.Answer;
+
+/**
+ * This data provider asks a user about security teams.
+ */
+public class AskAboutSecurityTeam<T> extends AbstractInteractiveDataProvider<T> {
+
+  @Override
+  protected AskAboutSecurityTeam<T> ask(T object, ValueSet values) {
+    String question = String.format(
+        "Does project '%s' have a security team? Say yes, no, or skip, please.", object);
```

Comment:
How is skip handled here? I mean in the switch case?

Reply:
Good point. I'll add a case for `SKIP`.